gdal/src/main/scala/geotrellis/contrib/vlm/gdal/GDALException.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.contrib.vlm.gdal
+
+
+/** Base class for all Exceptions involving GDAL. */
+class GDALException(message: String) extends Exception(message)
```

Comment:
Another  consideration, assuming all these exceptions are raised when getting error codes from the underlying GDAL library, is to have a field called `gdal_error_code` to capture the value returned.

Reply:
This is something that should be added once the return value reflects the number of the error thrown.

Reply:
@jbouffard I think this comment is still unaddressed.